packages/unified-selection/src/test/SelectionStorage.test.ts

```diff
@@ -0,0 +1,343 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as moq from "typemoq";
```

Comment:
`typemoq` is not actively maintained. Can you remove it? It looks like it is used for mocking `imodeConnection` but it seems that something like `const imodel = {} as IModelConnection` should do the trick.